.github/workflows/build.yml

```
@@ -186,32 +186,34 @@ jobs:
           fi
 
           echo ${PR_NUMBER} > result/pr_number
-      
+      - name: Compress build artifacts
+        run: |
+          tar -czvf packages.tar.gz -C repodata .  
```

```suggestion
          tar -czvf packages.tar.gz -C repodata .
```

nit: trailing whitespace